modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts

```diff
@@ -246,7 +246,7 @@ const setup = (extras: WindowManagerSetup): WindowManagerImpl => {
     return DialogManager.DialogManager.open<T>(factory, config);
   };
 
-  const openBottomInlineDialog = <T extends Dialog.DialogData>(config: Dialog.DialogSpec<T>, anchor: InlineDialogAnchor, closeWindow: (dialogApi: Dialog.DialogInstanceApi<T>) => void, ariaAttrs: boolean = false): Dialog.DialogInstanceApi<T> => {
+  const openBottomInlineDialog = <T extends Dialog.DialogData>(config: Dialog.DialogSpec<T>, anchor: InlineDialogAnchor, closeWindow: (dialogApi: Dialog.DialogInstanceApi<T>) => void, windowParams: WindowParams): Dialog.DialogInstanceApi<T> => {
```

Comment:
Should this have a default value for windowParams to make it an optional parameter, since ariaAttrs was?

Reply:
The default value of `ariaAttrs` is moved here, https://github.com/tinymce/tinymce/pull/8863/files#diff-a6300aae052657da2944a96441fce1034987e58bd3890c86a1b98f27a81f2b30R27.

Reply:
Missed that, thanks